smileydts/death-or-glory
# Dog 33 integrate sse

To test:

- Checkout the branch
- Import the Postman collection `dog.postman-collection.json` into Postman
- `docker-compose up`
- Navigate to localhost:5173
- Join the game with some name
- Observe that you're the only one in the game
- In Postman, join with three new players (i.e. run three API calls that are under the section "joins")
- After each join, observe that a new player is in the game
- In the console log, once the 4th player joins, observe a message saying that the connection to get players is closed
